src/main/java/biblioteket/roborally/actors/IRobot.java

```diff
@@ -0,0 +1,144 @@
+package biblioteket.roborally.actors;
+
+import biblioteket.roborally.Direction;
+import biblioteket.roborally.IPosition;
+
+public interface IRobot {
+
+    /**
+     * Set the player who is the owner of the robot.
+     * @param player
+     */
+    void setPlayer(IPlayer player);
+
+    /**
+     * Get the player who is the owner of the robot.
+     * @return the player who is the owner of the robot.
+     */
+    IPlayer getPlayer();
+
+    /**
+     * Set the number of damage the robot has taken.
+     * @param damageTokens
+     */
+    void setNumberOfDamageTokens(int damageTokens);
+
+    /**
+     * Return number of damage tokens a robot has.
+     * @return number of accumulated damage tokens
+     */
+    int getNumberOfDamageTokens();
+
+    /**
+     * Adds one damage token to the robot.
+     */
+    void addOneDamageToken();
+
+    /**
+     * Removes one damage token from the robot.
+     */
+    void removeOneDamageToken();
```

Comment:
I also feel that these are superfluous, as they are simply `removeDamageTokens(1)` etc.